app/views/layouts/application.html.erb

```diff
@@ -32,7 +36,7 @@
   <div class="header__inner">
     <h1>
       <a href="/">
-        <%= render 'layouts/title', mascot: current_user&.mascot, pervert: current_user&.pervert || false %>
+        <%= render 'layouts/title', mascot: mascot, pervert: current_user&.pervert || false %>
```

Comment:
I didn't touch the logic in layouts/title that defaults to Ki, since I figured it makes sense for that individual file to still have its own fallback case.